docs/destinations/forwarding-events/snowbridge/configuration/targets/http/index.md

```diff
@@ -3,6 +3,10 @@ title: "HTTP Target"
 description: "Send data over HTTP."
 ---
 
+:::note 
+Version 3.0.0 makes breaking changes to the http target. Details on migrating can be found [in the migration guide](/docs/destinations/forwarding-events/snowbridge/3-X-X-upgrade-guide/index.md)
+:::
+
 ## Basic Authentication
```

Comment:
the page is called HTTP target, but it starts by talking about basicauth. What's the connection?

Reply:
Basicauth is a means of authorising http. Every source and target section starts with a description of how to do auth - which was how @stanch wanted it when we first wrote the docs